net-vpn/i2pd/i2pd-2.49.0.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="I2P: End-to-End encrypted and anonymous Internet"
+DESCRIPTION="i2pd (I2P Daemon) is a full-featured C++ implementation of I2P client.\
+I2P (Invisible Internet Protocol) is a universal anonymous network layer.\
+All communications over I2P are anonymous and end-to-end encrypted, participants \
+don't reveal their real IP addresses.\
+I2P client is a software used for building and using anonymous I2P networks. \ 
+Such networks are commonly used for anonymous peer-to-peer applications \
+(filesharing, cryptocurrencies) and anonymous client-server applications \ 
+(websites, instant messengers, chat-servers).\
+I2P allows people from all around the world to communicate and share information without restrictions.
+"
+HOMEPAGE="https://i2pd.website/"
+COPYRIGHT="i2pd developers"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/PurpleI2P/i2pd/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="d8e07f78cc9f1ba65e8460db27c649dd0cfdd3ba334725f8d6f9ee815cb40e68"
+PATCHES="$portName-$portVersion.patchset"
+SOURCE_DIR="i2pd-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+boostMinumumVersion="1.83.0"
+
+PROVIDES="
+	i2pd$secondaryArchSuffix
+	cmd:i2pd$secondaryArchSuffix
+	lib:libi2pd$secondaryArchSuffix
+	lib:libi2pdclient$secondaryArchSuffix
+	lib:libi2pdlang$secondaryArchSuffix
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost_date_time$secondaryArchSuffix >= $boostMinumumVersion
+	lib:libboost_filesystem$secondaryArchSuffix >= $boostMinumumVersion
+	lib:libboost_program_options$secondaryArchSuffix >= $boostMinumumVersion
+	lib:libboost_system$secondaryArchSuffix >= $boostMinumumVersion
+	lib:libcrypto$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libboost_config$secondaryArchSuffix >= $boostMinumumVersion
+	devel:libboost_date_time$secondaryArchSuffix >= $boostMinumumVersion
+	devel:libboost_filesystem$secondaryArchSuffix >= $boostMinumumVersion
+	devel:libboost_program_options$secondaryArchSuffix >= $boostMinumumVersion
+	devel:libboost_system$secondaryArchSuffix >= $boostMinumumVersion
+	devel:libcrypto$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:linguist$secondaryArchSuffix >= 5
+	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	export LDFLAGS="-lnetwork -lbsd"
+	cmake -Sbuild -Bbuild $cmakeDirArgs \
+	-DCMAKE_BUILD_TYPE=Release \
+	-DBUILD_SHARED_LIBS=ON \
+	-DBoost_USE_STATIC_LIBS=OFF \
+	-DBoost_USE_STATIC_RUNTIME=OFF
```

Comment:
Use indents here:
```
	cmake -Sbuild -Bbuild $cmakeDirArgs \
		-DCMAKE_BUILD_TYPE=Release \
		-DBUILD_SHARED_LIBS=ON \
		-DBoost_USE_STATIC_LIBS=OFF \
		-DBoost_USE_STATIC_RUNTIME=OFF
```

Reply:
Missing "make -C build $jobARgs"

Reply:
Shouldn't be `-S.` ?

Reply:
CMakeLists.txt and needed files are provided in "build" directory. :)

Reply:
Then probably another directory name should be used as build directory. With cmake, a separate build directory is usually preferred.